packages/builders/src/messages/formatters.ts

```diff
@@ -11,12 +12,21 @@ export function codeBlock<C extends string>(content: C): `\`\`\`\n${C}\n\`\`\``;
 /**
  * Wraps the content inside a codeblock with the specified language
  *
- * @param language - The language for the codeblock
  * @param content - The content to wrap
+ * @param language - The language for the codeblock
  */
-export function codeBlock<L extends string, C extends string>(language: L, content: C): `\`\`\`${L}\n${C}\n\`\`\``;
-export function codeBlock(language: string, content?: string): string {
-	return typeof content === 'undefined' ? `\`\`\`\n${language}\n\`\`\`` : `\`\`\`${language}\n${content}\n\`\`\``;
+export function codeBlock<C extends string, L extends string>(content: C, language: L): `\`\`\`${L}\n${C}\n\`\`\``;
+export function codeBlock(content: string, language?: string): string {
+	if (language && language.length > 18) {
```

Comment:
Whats this `18` based on again?
I imagine its the longest possible language string but can we put a comment there or something? So we don't get confused at one point.

Reply:
It's based on this message: https://canary.discord.com/channels/222078108977594368/1001614734203834499/1003131709803483156
I'll add that comment